dts/bindings/clock/atmel,sam0-dfll.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2019 Derek Hageman <hageman@inthat.cloud>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: SAM0 Digital Frequency Locked Loop (DFLL)
+version: 0.1
+
+description: >
+    This is a representation of a SAM0 fixed frequency Digital Frequency
+    Locked loop.  This is a fixed frequency clock that can optionally use
+    a reference input to adjust its calibration dynamically.
+
+inherits:
+    !include clock.yaml
+
+properties:
+    compatible:
+      constraint: "atmel,sam0-dfll"
+
+    clock-frequency:
```

Comment:
What is `clock-frequency` suppose to mean?

Reply:
This is the output frequency of the oscillator.  It is determined by the hardware (e.g. 48 MHz on SAMD2x or 32 MHz on SAML1x).

I have clarified the description on the binding.